pages/services/spark/2.6.0-2.3.2/install/index.md

```diff
@@ -0,0 +1,376 @@
+## Customize {{ model.techShortName }} user
+DC/OS {{ model.techShortName }} user defaults to `nobody`, to override it set the following configuration variable:
+
+```json
+{
+  "service": {
+    "user": "<user>"
+  }
+}
+```
+
+{{ model.techShortName }} runs all of its components in Docker containers. Since the Docker image contains a full Linux userspace with
+its own `/etc/users` file, it is possible for the user `nobody` to have a different UID inside the
+container than on the host system. Although user `nobody` has UID 65534 by convention on many systems, this is not
+always the case. As Mesos does not perform UID mapping between Linux user namespaces for Docker containerizer,
+specifying a service user of `nobody` in this case will cause access failures when the container user attempts to open or execute a filesystem
+resource owned by a user with a different UID, preventing the service from launching. If the hosts in your cluster
+have a UID for `nobody` other than 65534, you will need to provide valid UID for `nobody` in the configuration to run the service
+successfully. For example, on RHEL/Centos based distributions:
+
+```json
+{
+  "service": {
+    "user": "nobody",
+    "docker_user": "99"
+  }
+}
+```
+
+<a name="custom-network"></a>
+## Configure {{ model.techShortName }} virtual network
+DC/OS {{ model.techShortName }} can be launched in a virtual network and configured with network labels.
+
+Here's an example of {{ model.techShortName }} configuration for DC/OS overlay network:
+
+```json
+{
+  "service": {
+    "virtual_network_enabled": true,
+    "virtual_network_name": "dcos",
+    "virtual_network_plugin_labels": [
+        {"key": "key_1", "value": "value_1"},
+        {"key": "key_2", "value": "value_2"}
+    ]
+  }
+}
+```
+
+When DC/OS {{ model.techShortName }} deployed in virtual network, all submitted jobs will run in the same
+network until another network is specified in job submit arguments.
+
+Please check existing limitations of virtual network support [here][20].
+
+# Minimal installation
+
+For development purposes, you can install {{ model.techShortName }} on a local DC/OS cluster. For this, you can use [dcos-vagrant][16].
+
+1. Install DC/OS Vagrant:
+
+	Install a minimal DC/OS Vagrant according to the instructions [here][16].
+
+1. Install {{ model.techShortName }}:
+
+   ```bash
+   dcos package install spark
+   ```
+
+1. Run a simple job:
+
+   ```bash
+   dcos spark run --submit-args="--class org.apache.spark.examples.SparkPi https://downloads.mesosphere.com/spark/assets/spark-examples_2.11-2.3.2.jar 30"
+   ```
+
+<p class="message--important"><strong>IMPORTANT: </strong>A limited resource environment such as DC/OS Vagrant restricts some of the features available in DC/OS {{ model.techName }}.  For example, unless you have enough resources to start up a 5-agent cluster, you will not be able to install DC/OS HDFS, and you thus won't be able to enable the history server.</p>
```

Comment:
Rewrite line #164 
For example, you must have enough resources to start up a 5-agent cluster, otherwise you will not be able to install DC/OS HDFS an enable the history server.